javalosjr96/liferay-portal
# LPD-28343

Here are the first batch of tests from DatabaseUpgradeClient.  

To run these tests locally:
1. Launch the portal to setup a default database.
2. Add the properties for the database upgrade tool.
3. Navigate to modules/tests/playwright
    - run _npm install_
4. Run playwright test
    - run _npm run playwright -- test -g 'CheckUpgradeClientAdditionalSettings'_
5.  The test should run and display the ant build and playwright should pass/fail the test.

The bottom four are mainly there to test loading the properties through the environment.  The ant build does recognize the value, however the expect file needs to be updated.
